spec/API_specification/array_api/elementwise_functions.py

```diff
@@ -1014,25 +1062,66 @@ def floor(x: array, /) -> array:
     - If ``x_i`` is ``+0``, the result is ``+0``.
     - If ``x_i`` is ``-0``, the result is ``-0``.
     - If ``x_i`` is ``NaN``, the result is ``NaN``.
+    """
+
+
+def floor_divide(x1: array, x2: array, /) -> array:
```

Comment:
`floor_divide` ended up with duplicate content

Reply:
Thanks for catching. Fixed.